SparkPost/2web2ui
# EO-852: Added RV price modal for RV page

### What Changed
 - Added button to link to price modal
 - Added price modal

### How To Test
 - Navigate to `/recipient-validation`
 - Check that button on top right says `See Pricing`
 - Check that it opens modal

### To Do
